PWGHF/vertexingHF/AliAnalysisTaskSEXic0pp13TeV.cxx

```diff
@@ -0,0 +1,1514 @@
+    DefineOutput (3, TTree::Class());
+    DefineOutput (4, TTree::Class());
+    DefineOutput (5, TTree::Class());
+    DefineOutput (6, TTree::Class());
+    DefineOutput (7, AliNormalizationCounter::Class());
+    fMCCutTreeVariable = 0;
+    fPaireXiTreeVariable = 0;
+    fMCTreeVariable = 0;
+    fEventTreeVariable = 0;
+    fMCCutTree = 0;
+    fPaireXiTree = 0;
+    fMCTree = 0;
+    fEventTree = 0;
+}
+//___________________________________________________________________
+AliAnalysisTaskSEXic0pp13TeV::AliAnalysisTaskSEXic0pp13TeV( const AliAnalysisTaskSEXic0pp13TeV& ap ) :
+  fOption(ap.fOption) {;}
+
+//___________________________________________________________________
+AliAnalysisTaskSEXic0pp13TeV& AliAnalysisTaskSEXic0pp13TeV::operator = ( const AliAnalysisTaskSEXic0pp13TeV& ap ) {
+  // assignment operator
+  this->~AliAnalysisTaskSEXic0pp13TeV();
+  new(this) AliAnalysisTaskSEXic0pp13TeV(ap);
+  return *this;
+}
+//___________________________________________________________________
+AliAnalysisTaskSEXic0pp13TeV::~AliAnalysisTaskSEXic0pp13TeV(){
+  delete fOutput;
+  delete fMCCutTree;
+  delete fPaireXiTree;
+  delete fMCTree;
+  delete fTrackCuts;
+  delete fPIDResponse;
+  delete fRunTable;
+  delete fCounter;
+  delete fEvtCuts;
+  delete fMCCutTreeVariable;
+  delete fPaireXiTreeVariable;
+  delete fMCTreeVariable;
+  delete fEventTreeVariable;
+}
+
+//___________________________________________________________________
+void AliAnalysisTaskSEXic0pp13TeV::UserCreateOutputObjects(){
+  fTrackCuts = new AliESDtrackCuts();
+  {
+    //fTrackCuts->SetMinNClustersTPC(fNClustersTPCMin);  //tpc cluster
+    fTrackCuts->SetRequireTPCRefit(kTRUE);  //tpc refit
+    fTrackCuts->SetRequireITSRefit(kTRUE);  //its refit
+    fTrackCuts->SetMinNClustersITS(fNClustersITSMin);  //its cluster
+    fTrackCuts->SetClusterRequirementITS(AliESDtrackCuts::kSPD, AliESDtrackCuts::kBoth);
+    fTrackCuts->SetMaxDCAToVertexXY(1.0);
+    fTrackCuts->SetMaxDCAToVertexZ(2.0);
+    fTrackCuts->SetDCAToVertex2D(kTRUE);
+    fTrackCuts->SetMaxChi2PerClusterITS(36);
+  }
+
+  fEvtCuts=new AliRDHFCutsXictoeleXifromAODtracks();
+
+  fEvtCuts->SetUsePhysicsSelection(kTRUE);
+  fEvtCuts->SetTriggerClass("");
+  fEvtCuts->SetUseInt7TriggerPP2012();
+  //fEvtCuts->ResetMaskAndEnableMBTrigger();
+
+  fHistos = new THistManager("histogram");
+
+  vector <TString> Event ={"All","ProperB","RDHCCutsSel","Sel","MB","PSpileup","Goodz","Goodzcut"};
+  auto hEventCount = fHistos->CreateTH1("hEventNumbers","",Event.size(), 0, Event.size());
+  for(auto i=0u; i<Event.size(); i++) hEventCount->GetXaxis()->SetBinLabel(i+1,Event.at(i).Data());
+
+  Double_t bin[8] = {1,2,3,4,5,6,8,12};
+  Double_t bin_old[8] = {0,1,2,3.2,4.4,6,8,12};
+  Double_t rapbin[21] = {-1,-0.9,-0.8,-0.7,-0.6,-0.5,-0.4,-0.3,-0.2,-0.1,0,0.1,0.2,0.3,0.4,0.5,0.6,0.7,0.8,0.9,1};
+  Double_t widebin[10] = {1,2,3,4,5,6,8,12,16,20};
+
+  fHistos->CreateTH1("NumOfEvtperRun","",5000,0,5000,"s");
```

Comment:
I am not sure about the convenience of using `THistManager::CreateTH1` instead of using the standard ROOT constructors. In any case, if you want to keep it, you should add the `PWGTools` library in the dependencies in the [PWGHF/vertexingHF/CMakeLists.txt](https://github.com/JinjooSeo/AliPhysics/blob/master/PWGHF/vertexingHF/CMakeLists.txt) file